### PR TITLE
cleaning code

### PR DESCRIPTION
**Pull request by ArtificialOwl**
- avoid moving null[] around,
- cleaning code

### CODE REVIEW

File: lib/Platform/ElasticSearchPlatform.php [hunk 1]
```diff
@@ -388,44 +388,34 @@ private function cleanHost(string $host): string {
...
-	}
+        try {
+            $hosts = array_map([$this, 'cleanHost'], $hosts);
+            $cb = ClientBuilder::create()
+                ->setHosts($hosts)
+                ->setRetries(3);
+            $this->configureAuthentication($cb, $hosts);
+
+            $this->client = $cb->build();
+        } catch (Exception $e) {
```
Comment (susnux) [inline lib/Platform/ElasticSearchPlatform.php:399]: Would it not be better to catch for `Throwable`? (While in this case it does not matter as it is thrown right again).

**Review by come-nc — APPROVED** [review]

File: lib/Platform/ElasticSearchPlatform.php (hunk 1 above)
Comment (ArtificialOwl) [reply lib/Platform/ElasticSearchPlatform.php]: you're right; removed the useless catch here and there

**Pull request merged**